pkg/services/sqlstore/migrations/libraryelements.go

```diff
@@ -61,4 +61,11 @@ func addLibraryElementsMigrations(mg *migrator.Migrator) {
 
 	mg.AddMigration("alter library_element model to mediumtext", migrator.NewRawSQLMigration("").
 		Mysql("ALTER TABLE library_element MODIFY model MEDIUMTEXT NOT NULL;"))
+
+	mg.AddMigration("add library_element folder uid", migrator.NewAddColumnMigration(libraryElementsV1, &migrator.Column{
+		Name: "folder_uid", Type: migrator.DB_NVarchar, Length: 40, Nullable: false, Default: "''",
+	}))
+
+	mg.AddMigration("populate library_element folder_uid", migrator.NewRawSQLMigration("").
+		Default("UPDATE library_element SET folder_uid = (SELECT uid FROM dashboard WHERE id = library_element.folder_id)"))
```

Comment:
@diegommm currently we cannot because the `library_element.folder_id` refers to the `dashboard` table sequential ID that difference from the `folder`'s table sequential ID.

Reply:
In that case I would suggest:
```SQL
UPDATE library_element
    SET folder_uid = (
        SELECT dashboard.uid
            FROM dashboard, folder
            WHERE
                dashboard.uid = folder.uid
                AND dashboard.id = library_element.folder_id
    )
```
So that we don't propagate potential cases of referential inconsistencies, given that we don't use foreign keys, wdyt?

Reply:
ok but in that case I think that we should include also the `org_id` column in the where clause so that that we use the index on (`org_id`, `uid`) columns.